src/api/client/cache.ts

```diff
@@ -44,7 +44,10 @@ const cache = new InMemoryCache({
             // if offset and limit are not provided.
             const offset = args?.offset ?? 0
             const limit = args?.limit ?? existing?.length
-            return existing?.slice(offset, offset + limit)
+            const sortingASC = args?.orderBy === VideoOrderByInput.CreatedAtAsc
+            return sortingASC
+              ? existing?.slice(offset, offset + limit)
+              : existing?.slice(offset, offset + limit).reverse()
```

Comment:
I'm not sure if that's fully working, seems it's not covering the case when we have 2 pages of videos

Reply:
True, I think the reverse should come before the slice here